kafka/src/main/java/io/specmesh/kafka/Provisioner.java

```diff
@@ -209,17 +257,22 @@ private static String readSchemaContent(final Path schemaPath) {
     /**
      * Provision acls in the Kafka cluster
      *
+     * @param validateMode for mode of operation
      * @param apiSpec the api spec.
      * @param adminClient th admin client for the cluster.
      * @return status of provisioning
      * @throws ProvisioningException on interrupt
      */
-    public static AclStatus provisionAcls(final KafkaApiSpec apiSpec, final Admin adminClient) {
-        final var aclStatus = AclStatus.builder();
+    public static ProvisionStatus.Acls provisionAcls(
+            final boolean validateMode, final KafkaApiSpec apiSpec, final Admin adminClient) {
```

Comment:
As above, I'd make this private and use the builder style API

Reply:
ill do the builder stuff in the next PR